RetailCoder.VBE/API/ParserState.cs

```diff
@@ -28,30 +77,28 @@ public interface IParserState
     [ComVisible(true)]
     [Guid("3D8EAA28-8983-44D5-83AF-2EEC4C363079")]
     [InterfaceType(ComInterfaceType.InterfaceIsIDispatch)]
-    public interface IParserStateEvents
+    public interface IRubberduckParserStateEvents
     {
-        void OnParsed();
-        void OnReady();
-        void OnError();
+        void OnStateChanged(ParserState state);
```

Comment:
Because I don't like the idea of implementing a zillion events....  And, I'm not sure--we explicitly invoke the event on the UI thread.